### DISCUSSION TITLE
Are snapshot builds published to an publicly accessible Maven repo?

### DISCUSSION THREAD

**Discussion by crankydillo** [resteasy/resteasy#3630] [Q&A] [answered]
An 'effective' pom for this project, led me to [this Maven repo](https://repository.jboss.org/nexus/content/repositories/snapshots/org/jboss/resteasy/resteasy-bom/), but I don't see any recent snapshot builds.  We'd like to test with 5.0.7.Final-SNAPSHOT without doing a local build.

Thanks!

**Comment by jamezp** [comment, marked as answer]
We don't typically upload SNAPSHOTS. I'm out of the office today, but I could look into it at some point for sure.
